api/v1alpha2/sharedazuredbs_types.go

```diff
@@ -50,6 +51,7 @@ const (
 	PSQLMemoryOptimized SkuTier = "MemoryOptimized"
 )
 
+// +kubebuilder:validation:Enum=Enabled;Disabled
```

Comment:
minor: You have two formats here, the shorter/simpler one and the longer/more verbose one. Feels like we should have a single format we prefer?

In most of the existing ASO code it seems to be the `Enabled;Disabled` format, but the generator is doing `{"Enabled","Disabled"}`. Unsure which we should pick but feels like 1 style per file is better?

Reply:
This syntax works, and I didn't get the other syntax to work in my testing - so I'd prefer to leave this as is.

Reply:
(Which is not to say the more verbose syntax is wrong, maybe I had a local issue.)

Reply:
What about the more verbose syntax you use above?
// +kubebuilder:validation:Enum={"Basic","GeneralPurpose","MemoryOptimized"}